packages/gatsby-telemetry/src/telemetry.ts

```diff
@@ -0,0 +1,353 @@
+const uuidv4 = require(`uuid/v4`)
+const EventStorage = require(`./event-storage`)
+const { cleanPaths } = require(`./error-helpers`)
+const { isCI, getCIName } = require(`gatsby-core-utils`)
+const os = require(`os`)
+const { join, sep } = require(`path`)
+const isDocker = require(`is-docker`)
+const showAnalyticsNotification = require(`./showAnalyticsNotification`)
+const lodash = require(`lodash`)
```

Comment:
can we change these all to module imports? typescript can't follow types from modules with require. You have to use `import` to get their types

Reply:
Converted most of them. There is some more work to be done to fully type this but I wanted to start by getting this over to TS to make future development easier